tests/utils/test_logging.py

```diff
@@ -24,3 +25,24 @@ def test_get_logger() -> None:
     assert isinstance(logger, Logger)
     assert log_file == str(expected_log_file)
     assert logger.name == __name__
+
+def test_reocurring_messages() -> None:
```

Comment:
typo

Reply:
oh yeah haha

Reply:
i knew something looked off!

Reply:
https://www.merriam-webster.com/words-at-play/difference-between-recur-and-reoccur-definitions or maybe not